crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -5135,6 +5135,10 @@ impl<'a> Checker<'a> {
                         _ => continue,
                     };
 
+                    if self.settings.dummy_import_rgx.is_match(full_name) {
+                        continue;
+                    }
+
```

Comment:
Nit: I recommend moving the line below the line 5152. The `ImportFrom` check should be cheaper than testing the name.

Reply:
This's a better idea

Reply:
But what if I want import like this?
If I just want to ignore _Name:

```python
from ImportFrom import Name as _Name
```
or
```python
from ImportFrom import OtherName, _Name
```